index.bs

```diff
@@ -2468,6 +2520,61 @@ devices=] that are rarely used or do not include a [=platform authenticator=]; o
 backup [=public key credential|credentials=] in case another [=authenticator=] is lost.
 
 
+### Storage modality ### {#sctn-authenticator-storage-modality}
+
+An [=authenticator=] can store a [=credential private key=] in one of two ways:
+
+ 1. In persistent storage embedded in the [=authenticator=], [=client=] or [=client device=], e.g., in a secure element.
+
+    A [=credential private key=] stored in this way is a [=client-side-resident credential private key=].
+
+ 1. By encrypting the [=credential private key=] such that only this [=authenticator=] can decrypt it and letting the resulting
+    ciphertext be the [=credential ID=] for the [=public key credential source=]. The [=credential ID=] is stored by the [=[RP]=]
+    and returned to the [=authenticator=] via the {{PublicKeyCredentialRequestOptions/allowCredentials}} option of
+    {{CredentialsContainer/get()}}, which allows the [=authenticator=] to decrypt and use the [=credential private key=].
+
+    This enables the [=authenticator=] to have unlimited storage capacity for [=credential private keys=], since the encrypted
+    [=credential private keys=] are stored by the [=[RP]=] instead of by the [=authenticator=] - but it means that a
+    [=credential=] stored in this way must be retrieved from the [=[RP]=] before the [=authenticator=] can use it.
+
+Which of these storage strategies an [=authenticator=] supports defines the [=authenticator=]'s <dfn>storage modality</dfn> as
+follows:
+
+- An [=authenticator=] has the <dfn>local storage modality</dfn> if it supports storing [=credential private keys=] in persistent
+    storage embedded in the [=authenticator=], or if it supports both storage strategies. An [=authenticator=] with [=local
+    storage modality=] is also called <dfn>local storage capable</dfn>.
+
+- An [=authenticator=] has the <dfn>remote storage modality</dfn> if it does not have the [=local storage modality=], i.e., it
+    only supports storing [=credential private keys=] as a ciphertext in the [=credential ID=].
+
+Note that an [=authenticator=] with [=local storage modality=] MAY support both storage strategies. In this case, the
+[=authenticator=] MAY at its discretion use different storage strategies for different [=public key credential|credentials=],
+though subject to the {{AuthenticatorSelectionCriteria/requireResidentKey}} option of {{CredentialsContainer/create()}}.
+
+
+### <dfn>Authentication factor capability</dfn> ### {#sctn-authentication-factor-capability}
+
+There are three broad classes of [=authentication factors=] that can be used to prove an identity during an [=authentication
```

Comment:
prove an identity -> [=user verification|verify a user=]

Reply:
I think that would be a misnomer in this case, since UV is a specific term for one way to implement some kinds of authentication factor, but "prove an identity" in this sentence is meant to be about the general problem of how to prove the authenticity of a claim of identity.